spiketools/tests/sim/test_params.py

```diff
@@ -0,0 +1,127 @@
+"""Tests for spiketools.sim.params"""
+
+from spiketools.sim.params import *
+
+###################################################################################################
+###################################################################################################
+
+def test_upd_height():
+    """Test upd_height"""
+    params = {'height_mean' : 5, 'width_mean' : 5, 'noise_std' : 1, 'place_loc_std' : 1, 'skewness_mean' : 1, 'presence_ratio' : 1, 'base_mean' : 1, 'n_trials' : 10}
```

Comment:
this set of params gets defined and used a lot, which is basically alot of copied code. For this purpose there are pytest fixtures. You can define a fixture in `spiketools/tests/conftest.py` - in which you define a function that defines and yields a variable of interest, with a fixture decorator. You can then pass this variable to a test function - for example, something like:
`def test_upd_width(t_place_sim_param):`
^if `t_place_sim_params` is the name of your fixture. 

Check through examples in the module of tests using fixtures for examples - let me know if anything doesn't seem to work and we can chat more on this